test/daml/Test/Lifecycle.daml

```diff
@@ -252,22 +248,22 @@ testEuropeanCall = script do
   t <- getDate
 
   -- Exercise is a no-op
-  remaining <- Lifecycle.exercise observe25 (bearer, zero) option t
+  remaining <- Lifecycle.exercise observe25 (bearer, zero) option today t
   remaining === option
 
   -- and so is settlement
-  Lifecycle.Result{pending, remaining} <- Lifecycle.lifecycle observe25 option t
+  Lifecycle.Result{pending, remaining} <- Lifecycle.lifecycle observe25 option today t
   remaining === option
   pending === []
 
   -- At maturity
   setDate tomorrow
   t <- getDate
 
-  remaining <- Lifecycle.exercise observe25 (bearer, payoff) option t
+  remaining <- Lifecycle.exercise observe25 (bearer, payoff) option today t
```

Comment:
With this extra parameter these tests become kind of hard to read with `today`/`tomorrow` being quite confusing. Mainly because we're writing them in an imperative way (`setDate xxx`). Not an issue for now, but we may want to consider renaming today, tomorrow etc. to t0 t1 . Perhaps that would be clearer.

Reply:
agree